ext/hash/hash.c

```diff
@@ -345,23 +345,14 @@ PHP_HASH_API int php_hash_unserialize(php_hashcontext_object *hash, zend_long ma
 
 /* Userspace */
 
-static void php_hash_do_hash(INTERNAL_FUNCTION_PARAMETERS, int isfilename, zend_bool raw_output_default) /* {{{ */
-{
-	zend_string *digest, *algo;
-	char *data;
-	size_t data_len;
-	zend_bool raw_output = raw_output_default;
+static void php_hash_do_hash(
+	zval *return_value, zend_string *algo, char *data, size_t data_len, zend_bool raw_output, int isfilename
```

Comment:
```suggestion
	zval *return_value, zend_string *algo, char *data, size_t data_len, zend_bool raw_output, bool isfilename
```
Maybe?

Reply:
Makes sense for me, so I applied this change to master.